documents/operating.md

```diff
@@ -41,13 +41,101 @@ SECTION 3: 🤔🤔🤔 Projects 🤔🤔🤔
 ---------------------------------------------------------------------
 SECTION 4: 💵💵💵 MONEY 💵💵💵
 ---------------------------------------------------------------------
-- [ ] Capital contributions
-- [ ] Allocating resources to projects
-- [ ] Distributing profits
-- [ ] Compensation
-- [ ] Expenses
-- [ ] Taxes
-- [ ] Company canoe
+- [x] Capital contributions
+- [x] Allocating resources to projects
+- [x] Distributing profits
+- [x] Compensation
+- [x] Expenses
+- [x] Taxes
+- [x] Company canoe
+
+
+### Capital Contributions
+Corporate Overlords must contribute a $300 initial capital contribution
+to BIFFUD within three months of indoctrination.
+
+The Hive Mind can vote to increase this number, as described in Section 5.
```

Comment:
Increase which number?  The 300$ capital contribution?  the 3 month window of time?  Both?